update/genesis/import.go

```diff
@@ -285,18 +293,18 @@ func (si *stateImport) importDataTrie(fileName string, shID uint32) error {
 		if err != nil {
 			return err
 		}
-		si.tries[fileName] = dataTrie
-		si.reader.CloseFile(fileName)
+		si.tries[identifier] = dataTrie
+
 		return nil
 	}
 
-	for {
-		key, value, err = si.reader.ReadNextItem(fileName)
+	for _, key := range keys {
+		value, err = si.hardforkStorer.Get(key)
 		if err != nil {
 			break
```

Comment:
return err directly. no need for break and if after for

Reply:
I wanted to do a general error wrapping at the end of the function